app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -40,6 +41,10 @@ class AdvancedFilter extends React.Component {
       }
     });
 
+    if (advancedFilterOptions.filter(advancedFilter => advancedFilter.name === 'vaccination').length === 0) {
```

Comment:
is there a reason this check is necessary since this is only called in the `componentDidMount` where it should not exist in the options?

Reply:
@sgober Without this check, the Vaccination (Combination) filter was being added multiple times. When I used the Vaccination (Combination) filter on the Patients Table and then created a Custom Export using the same Vaccination (Combination) filter, it appears in the dropdown multiple times without the check (see screenshot), 

<img width="1122" alt="Screen Shot 2022-03-10 at 10 17 03 AM" src="https://user-images.githubusercontent.com/2328582/157694396-6d2ccb38-e0f7-4d9f-9850-fc779008ce23.png">

The result of `getVaccineAdvancedFilters(this.props.vaccine_standards)` is added to the advancedFilterOptions every time `componentDidMount `is called without the check. 

If there is a better way to handle this, let me know. Thanks!

Reply:
I don't think it's worth putting more effort into this because it will be removed with the completion of [1636](https://tracker.codev.mitre.org/browse/SARAALERT-1636)